### DISCUSSION TITLE
все работало хорошо первые пару попыток, но...

### DISCUSSION THREAD

**Discussion by YourFriend8890** [MrZillaGold/VK2Discord#753] [Q&A]
![image](https://github.com/MrZillaGold/VK2Discord/assets/107394655/8251d14f-c590-472e-bbc3-fd8d414c39e4)
мне надо было сменить аватар и настроить время, как вдруг, эта штука откинулась на 100 ошибку и все. что делать, чтобы это починить? пробовал по новой все делать - 0 эффекта.

**Comment by egorprnn** [comment]
Перегенерируйте токены от страниц используя [ссылку](https://oauth.vk.com/authorize?client_id=2685278&scope=1073737727&redirect_uri=https://oauth.vk.com/blank.html&display=page&response_type=token&revoke=1)
